Bugfix/nginx

vc0sta opened the pull request.

vcosta-gls committed:
Fix nginx creating files as directories

vcosta-gls committed:
Set default ansible inventories

vcosta-gls committed:
Changing some tags

RCristiano commented on ansible/roles/prerequisites/tasks/main.yml:
```diff
@@ -45,6 +45,7 @@
   with_items: "{{ services }}"
   tags:
     - prerequisites
+    - nginx
```
Não entendi o motivo dessa tag nessa task?

vc0sta replied:
é que eu tava com clean install, e se rodo só o nginx, ele n tava criando todas as deps do serviço

RCristiano committed:
🚑 Fix order that conf is generated and nginx up

Nginx needs to be active after generating the nginx.conf file
Otherwise the container mounts the file path as a directory

RCristiano commented:
Fiz uma alteração sensível no 754b58f

Testei diversas vezes a correção removendo todo o diretório em `/tmp/*` e percebi que quando o container subia ele gerava uma pasta `nginx.conf` onde deveria ser um arquivo, porque ele não encontrava o path para fazer o mapeamento do volume.
Percebi que todos os containers subiam antes de o arquivo ser gerado, então mudei a ordem para que só subissem posteriormente. Também removi a task que reiniciava o nginx, já que ela executaria logo após os containers subirem, inclusive o nginx.
